src/components/utils.ts

```diff
@@ -607,6 +607,30 @@ export function getValidUrl(url: string): string {
   }
 }
 
+/**
+ * Creates a UUID v4 version
+ *
+ * @returns {string}
+ */
+function uuidv4(): string {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+    // tslint:disable: no-bitwise
+    // tslint:disable-next-line: triple-equals
+    const r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
+
+    return v.toString(16);
+  });
+}
+
+/**
+ * Create a block id
+ *
+ * @returns {string}
+ */
+export function generateBlockId(): string {
+  return uuidv4();
```

Comment:
Check this comment please https://github.com/codex-team/editor.js/pull/1493#issuecomment-773294522

Reply:
Ok i will change to nano(10)

Reply:
@cobbcheng need any help on this?